test/quantization/core/test_workflow_module.py

```diff
@@ -1102,6 +1108,47 @@ def test_fused_mod_reduce_range(self):
         self.assertEqual(obs.quant_min, 0)
         self.assertEqual(obs.quant_max, 127)
 
+    def test_embedding_bag_qat_config(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.emb1 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                self.emb2 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+
+            def forward(self, indices):
+                return torch.cat((self.emb1(indices), self.emb2(indices)))
+
+        model = Model()
+        indices = torch.randint(0, 10, (5, 12))
+        qat_mappings = get_default_qat_module_mappings()
+        qat_mappings[nn.EmbeddingBag] = torch.nn.qat.EmbeddingBag
```

Comment:
can we change the default qat mappings instead of expecting the users to do this?

Reply:
I'm trying to be defensive, prior to fully implementing the embedding QAT workflow, in order to avoid any breaking changes.  Do you think some thing like adding `get_embedding_qat_module_mappings()` would be a good in-between?